userguide/tn_initial.rst

```diff
@@ -274,17 +274,18 @@ Enter the default password of *abcd1234*.
    :guilabel:`Password confirmation` fields, and click :guilabel:`OK`
    to save the new password to use on subsequent logins.
 
-On the first login, the EULA found in :ref:`Appendix A` is displayed,
-along with a box where the license key for the %brand% array can be
-pasted. Read the EULA, paste in the license key, then click
-:guilabel:`I agree` to accept the EULA and apply the license.
+On the first login, the End User License Agreement (EULA) found in
+:ref:`Appendix A` is displayed, along with a box where the license key
+for the %brand% array can be pasted. Read the EULA, paste in the license
+key, then click :guilabel:`I agree` to accept the EULA and apply the
```

Comment:
```suggestion
key, then click :guilabel:`I agree`.

Reply:
Just a suggestion; Feels a bit more concise this way.

Reply:
Agreed.